### PR TITLE
React Capstone Project: Emoji Website

### PR DESCRIPTION
**Pull request by peter-akhigbe**
The project includes the following features:

- A list of items that can be filtered by some parameters, such as country name.
- A detail page for each item.
- A video presenting the features of the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+- **React Router**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo & Presentation <a name="live-demo"></a>
+
+- [Live Demo Link](https://react-capstone-emoji-website.onrender.com/)
+- [Presentation Link](https://www.loom.com/share/0d6b0f70c8ec4083b5c092a1c414c552/)
```
Comment (DammyShittu) [inline README.md:66]: - [ ] Kindly remake your video presentation demonstrating different parts of your app by navigating across the pages.

- [ ] In your video demo be sure to highlight your code in the editor that you are proud of or find interesting.

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+            </Link>
+          );
+        })}
+      </ul>
+    </div>
+  );
+};
+
+export default Home;
```
Comment (DammyShittu) [inline src/pages/Home.js:50]: - [ ] According to the requirements we should maintain the space between the elements and the page components. The intro section should have similar spacing as the one on the design. Kindly improve the look of the banner section (the part containing "Emojis") to match the one on the design (the section containing "Europe").

- [ ] Kindly add some descriptive images (or icons) to your items on the home page as we have on the design. If you have no images then a generic placeholder image that fits the app story would do.

- [ ] Kindly add the alternating color pattern as we have in the design.

  Your Implementation | Behance Design
  :---:|:---:
  mkdidd | kffifor

File: src/pages/Details.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useParams, Link } from 'react-router-dom';
+
+const Details = () => {
```
Comment (DammyShittu) [inline src/pages/Details.js:5]: - [ ] Kindly improve the styling on the details page to match the design. The values of the items should be move the right as we have on the design.

- [ ] There should be a back (<) button on the top left of the page for navigating back to the root page.

  Your Implementation | Behance Design
  :---:|:---:
  ffkfkfk | jfkfifr

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @peter-akhigbe,

## Highlights
✔️ Mobile first design

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse curriculum.
+
+<!-- FAQ (optional) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
```
Comment (DammyShittu) [inline README.md:154]: - [ ] The acknowledgment section should contain credit to Nelson Sakwa whose design was used for this project

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @peter-akhigbe, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**